src/main/cpp/include/utils/json_config.h

```diff
@@ -90,6 +92,10 @@ class JSONConfigBase
     //Lower and upper bounds of callset row idx to import in this invocation
     int64_t m_lb_callset_row_idx;
     int64_t m_ub_callset_row_idx;
+    //Vid mapping file
+    std::string m_vid_mapping_filename;
+    //callset mapping file - if defined in upper level config file
+    std::string m_callset_mapping_file;
```

Comment:
Renamed m_vid_mapping_filename to m_vid_mapping_file to match the JSON file field